include/etiss/DebugSystem.h

```diff
@@ -53,16 +53,90 @@
 #include "etiss/System.h"
 #include <fstream>
 
-namespace etiss
+#include <cstring>
+#include <iostream>
+#include <memory>
+
+template<typename T, typename... Args>
+std::unique_ptr<T> make_unique(Args&&... args)
```

Comment:
use this instead: https://github.com/tum-ei-eda/etiss/blob/master/include/etiss/make_unique.h